inc/namespace.php

```diff
@@ -69,6 +69,16 @@ function bootstrap() {
 
 	// Disable the admin_email verification interval.
 	add_filter( 'admin_email_check_interval', '__return_zero' );
+
+	// Performance enhancements.
+	// Comment pagination is always enabled.
+	add_filter( 'pre_option_page_comments', '__return_true', 999 );
+
```

Comment:
@roborourke I got couple of questions about this:

1. Is it ok to use `999` priority or there is another way to make sure that nothing else can modify this? If someone else adds a filter with grater priority they can change the value.

2. Is it sufficient to use short-circuit filter like this? Or does I have to actually store the new value in the DB as well, so if WP accesses that value directly in DB it's set correctly? I couldn't figure out what the best hook/action to use to do that.

Thanks!

Reply:
@dashaluna 

1. I think we can leave the priority as the default 10. If someone is filtering that value then it's by design and we shouldn't get in the way of that. This is really to protect site admins / users from setting things up badly.

2. Yes this is sufficient, it's very uncommon for the data to be accessed directly so not something I'd worry about. For the parts of core that apply these settings they always use `get_option()` so no need to overthink this :)

Reply:
Well the original ticket does say `force limit comments per page to 50 maximum`, so I'm being forceful 💪

Reply:
Fair point! I think using code is all the "force" we need in this case though. Ultimately you can't really force anything on a developer in WP land.